myblog/blog/tests.py

```diff
@@ -94,3 +94,35 @@ def test_logout(self, client, user_1):
         assert response.status_code == 200
         # Check Log out not in content
         assert 'Log out' not in response.content.decode()
+    
+@pytest.mark.usefixtures("user_1")
+class TestApiBlogPost:
+    """To test the api endpoint 
+    """
+    pytestmark = pytest.mark.django_db
+    
+    def test_blogpost_api(self, client, user_1):
+        
+        url = reverse('api/post')
+        
+        # Api GET
+        response = client.get(url,format = 'json')
+                
+        # Check response code
+        assert response.status_code == 200
+        
+        # Check db
+        posts = BlogPost.objects.count()
```

Comment:
I would put a more descriptive name to this variable, as `post_count` or something like that, or I would do the assert directly like `assert BlogPost.objects.count() ==  0` as you do it in the second assertion.